tests/test_http.py

```diff
@@ -5,7 +5,7 @@
 import responses
 import unittest
 
-from PyTado.const import CLIENT_ID, CLIENT_SECRET
+from PyTado.const import CLIENT_ID, CLIENT_ID_DEVICE, CLIENT_SECRET
```

Comment:
Should we clean up `CLIENT_ID` and `CLIENT_SECRET` from const as part of this refactor since they are not needed anymore?